modules/api-backups/classes/class-api-backups-3rd-party.php

```
@@ -534,75 +529,53 @@ public function render_api_backups_site( $website ) { //phpcs:ignore -- NOSONAR
-                            '<a href="admin.php?page=SettingsApiBackups">API Backups Settings</a>',
-                            '</br>',
-                            '<em>',
-                            '</em>',
-                            '<a href="admin.php?page=managesites&id=' . intval( $website_id ) . '">Child Site -> Edit</a>'
-                        );
-                    ?>
-                    </div>
-                </div>
-            <?php } else { ?>
+            \MainWP\Dashboard\MainWP_UI::render_empty_page_placeholder(
+                $title   = esc_html__( 'No API Backup Solution has been chosen.', 'mainwp' ),
```

PHPMD (2.15.0)

Avoid unused local variables such as '$title'. (Unused Code Rules)

(UnusedLocalVariable)